docs/download_files.md

```diff
@@ -46,6 +47,11 @@ The filtered counts file, `SCPCL000000_filtered.rds` is also an RDS file contain
 The cells in this file are those that remain after filtering using {ref}`emptyDrops <processing_information:filtering cells>`.
 As a result, this file only contains cell barcodes that are likely to correspond to true cells.
 
+The processed counts file, `SCPCL000000_processed.rds` is an RDS file containing a `SingleCellExperiment` object containing both the raw and normalized counts matrices.
+The filtered counts file is further filtered to remove any low quality cells, such as those with a low number of genes detected or high mitochondrial content.
+This file contains the raw and normalized counts data for only cell barcodes that have passed both levels of filtering.
+In addition to the normalized counts matrix, a selection of highly variable genes, and dimensionality reduction using both principal component analysis (PCA) and UMAP are available in the `SingleCellExperiment` object stored in the file.
```

Comment:
Small wording updates.

I removed the HVG section, as I think that may be confusing at this stage. When talking about the SCE contents, we can go into that level of detail, I think?

```suggestion
This file contains the raw and normalized counts data for cell barcodes that have passed both levels of filtering.
In addition to the counts matrices, the `SingleCellExperiment` object stored in the file includes the results of dimensionality reduction using both principal component analysis (PCA) and UMAP.
```